src/liballoc/collections/btree/map.rs

```diff
@@ -2378,6 +2378,34 @@ impl<'a, K: Ord, V> Entry<'a, K, V> {
         }
     }
 
+    #[unstable(feature = "or_insert_with_key", issue = "70996")]
```

Comment:
Can you create a proper tracking issue instead of just pointing to the PR?

Reply:
@Amanieu This is corrected in commit 3.